CMakeLists.txt

```diff
@@ -85,9 +85,17 @@ endif (NOT DATADIR)
 set (GSAD_DATA_DIR "${DATADIR}/gvm/gsad")
 set (GSAD_CONFIG_DIR "${SYSCONFDIR}/gvm/")
 
-if (NOT GVM_RUN_DIR)
-  set (GVM_RUN_DIR  "/run/gvm")
-endif (NOT GVM_RUN_DIR)
+if (NOT GSAD_RUN_DIR)
+  set (GSAD_RUN_DIR  "/run/gsad")
+endif (NOT GSAD_RUN_DIR)
+
+if (NOT GSAD_PID_PATH)
+  set (GSAD_PID_PATH     "${GSAD_RUN_DIR}/gsad.pid")
+endif (NOT GSAD_PID_PATH)
+
+if (NOT GVMD_RUN_DIR)
+  set (GVMD_RUN_DIR  "/run/gvmd")
+endif (NOT GVMD_RUN_DIR)
```

Comment:
It is required to allow setting the full path to the socket file because in gvmd is allowed to change the file name. Currently gsad requires that the socket must be `/foo/bar/gvmd.sock` but gvmd allows for `/foo/bar/baz`.

Please change this variable to `GVMD_PID_PATH` and use it as a full path to the socket file with the default `/run/gvmd/gvmd.sock` as in gvmd.

Reply:
`GVMD_PID_PATH` sets the path of the gvmd PID file, not the GMP socket.

For both gsad and gvmd the default socket is `gvmd.sock` in `$GVMD_RUN_DIR` and it is possible to set a custom path with command line options (`--munix-socket` in gsad and `--unix-socket` in gvmd.).